CYOAfeb21

```diff
@@ -0,0 +1,225 @@
+###Standard Question Setup###
+def multich(ch1="error1", ch2="error2", ch3="error3", ch4="error4"):
+	choice= input(f"""
+	Select Option:
+		A. {ch1}
+		B. {ch2}
+		C. {ch3}
+		D. {ch4}""")
+	return choice
+	
+###Questions for naming and selecting play class.###
+def givename():
+	name= input(print("Type your name."))
+	return name
+def giveclass():
+	playclass= multich(ch1='Warrior', ch2='Wizard', ch3='Cleric', ch4='Rogue')
+	return playclass
+	#add details about stats here
+#name, position, items, attack, defense, health, magic, experience, level, playclass
+
+###Define character class###
+class Character:
+	def __init__(self, name, position, items):
+		self.name= name
```

Comment:
The example I had technically hid the `name` field under `_name` and then added a property with a getter and a setter. What you have here doesn't need the getter or setter fields.

Reply:
What's the benefit of using getter/setter instead of open fields?